classes/log/EmailLogEntry.php

```diff
@@ -133,17 +171,18 @@ public function setAssocId($assocId)
      */
     public function getSenderFullName()
```

Comment:
How do you think, would implementing accessor would be better for cases like this? E.g., something like:
```php
public function senderFullName()
    {
        return new Attribute(
            get: fn () => ...
        );  
    }
```

Reply:
updated